src/Microsoft.Framework.Caching.SqlServer/SqlQueries.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    internal class SqlQueries
+    {
+        private const string TableInfoFormat =
+            "SELECT TABLE_CATALOG, TABLE_SCHEMA, TABLE_NAME, TABLE_TYPE " +
+            "FROM INFORMATION_SCHEMA.TABLES " +
+            "WHERE TABLE_SCHEMA = '{0}' " +
+            "AND TABLE_NAME = '{1}'";
+
+        private const string GetCacheItemFormat =
+            "SELECT Id, ExpiresAtTime, SlidingExpirationInTicks, AbsoluteExpiration, Value " +
+            "FROM {0} WHERE Id = @Id AND @UtcNow <= ExpiresAtTime";
```

Comment:
Consider putting back quotes around table name {0} because it is user-provided. http://stackoverflow.com/questions/11321491/when-to-use-single-quotes-double-quotes-and-backticks